### PR TITLE
[PAM][TR] PIM-4778: Display default thumbnail depending on file type

### PR DESCRIPTION
**Pull request by phaseinducer**
https://akeneo.atlassian.net/browse/PIM-4778

| Q                  | A
| ------------------ | ---
| Migration script?  | N
| Behats             | Y
| Specs & Static     | Y
| Changelog updated  | N

### CODE REVIEW

**Comment by nidup** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Controller/FileController.php [hunk 1]
```diff
@@ -20,48 +24,48 @@
  */
 class FileController extends Controller
 {
-    const DEFAULT_IMAGE_KEY = '__default__';
+    const DEFAULT_IMAGE_KEY = '__default__image__';
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Controller/FileController.php]: `__default_image__` ?

File: src/Pim/Bundle/EnrichBundle/Controller/FileController.php [hunk 2]
```diff
@@ -76,14 +80,23 @@ public function showAction(Request $request, $filename, $filter = null)
...
+                    return $this->imagineController->filterAction($request, $filename, $filter);
+                } catch (NotFoundHttpException $e) {
+                    return $this->renderDefaultImage(FileTypes::IMAGE, $filter);
+                }
+            }
+
+            return $this->renderDefaultImage($fileType, $filter);
         }
+
+        return $this->renderDefaultImage(FileTypes::MISC, $filter);
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Controller/FileController.php]: 4 `return` in a row is too much for me IMHO

**Comment by grena** [comment]
GTM

**Pull request merged**